test/extensions/tracers/datadog/span_test.cc

```diff
@@ -100,6 +100,22 @@ struct TestSetup {
   }
 };
 
+std::chrono::system_clock::time_point now() {
```

Comment:
Use TimeSource, or maybe SimulatedTimeSystem for this.

https://github.com/envoyproxy/envoy/blob/main/test/test_common/simulated_time_system.h